src/test/java/org/takes/misc/HrefTest.java

```diff
@@ -116,4 +116,20 @@ public void acceptsNonProperlyEncodedUrl() {
             Matchers.equalTo("http://www.netbout.com/%5Bfoo/bar%5D/read?file=%5B%5D%28%29.txt")
         );
     }
+
+    /**
+     * Href can build an URI with fragment.
+     */
+    @SuppressWarnings("PMD.AvoidDuplicateLiterals")
+    @Test
+    public void buildsUriWithFragment() {
```

Comment:
@msarhan there are two asserts inside one test method, please split them to two different methods (according to http://www.yegor256.com/2017/05/17/single-statement-unit-tests.html)